### PR TITLE
solo: Inject right environment in integration test

### PR DESCRIPTION
**Pull request by rohansingh**
Instead of looking at `DOCKER_HOST` and stuff from the environment of the
test runner, we need to be getting it from the environment of the host on
which the helios-solo job is run.

This fixes test failures when running `HeliosSoloIT` against a remote
Helios cluster (as opposed to a local helios-solo instance).

### CODE REVIEW

**Comment by davidxia** [comment]
LGTM. I'm also checking i can `mvn package` locally

**Comment by rohansingh** [comment]
Check out the latest update. I realized that `getEnvironment()` doesn't actually return what I think it does. It returns the `--env` flags that are set when starting the Helios agent, not the environment variables for the Helios agent itself.

So to get the Docker host and cert path from the agent, I'm now having it explicitly report those.

**Comment by rohansingh** [comment]
So much yak shaving.

File: helios-integration-tests/src/test/java/com/spotify/helios/HeliosSoloIT.java [hunk 1]
```diff
@@ -49,44 +51,53 @@
...
+        // using boot2docker, so use the unix socket endpoint
+        dockerHost = "unix:///var/run/docker.sock";
+        certPath = null;
+      }
+    }
 
     // build the helios-solo job
     final TemporaryJobBuilder solo = temporaryJobs.job()
-        .image(soloImage())
+        .image("registry.spotify.net/spotify/helios-solo:0.8.253")
```
Comment (davidxia) [inline helios-integration-tests/src/test/java/com/spotify/helios/HeliosSoloIT.java]: Why this change? Will the hardcoded version be a problem?

Comment (rohansingh) [inline helios-integration-tests/src/test/java/com/spotify/helios/HeliosSoloIT.java]: Oops.

File: helios-client/src/main/java/com/spotify/helios/common/descriptors/HostInfo.java [hunk 2]
```diff
@@ -94,7 +98,9 @@ public HostInfo(@JsonProperty("hostname") final String hostname,
                   @JsonProperty("memoryFreeBytes") final long memoryFreeBytes,
                   @JsonProperty("swapTotalBytes") final long swapTotalBytes,
                   @JsonProperty("swapFreeBytes") final long swapFreeBytes,
-                  @JsonProperty("dockerVersion") final DockerVersion dockerVersion) {
+                  @JsonProperty("dockerVersion") final DockerVersion dockerVersion,
```
Comment (davidxia) [inline helios-client/src/main/java/com/spotify/helios/common/descriptors/HostInfo.java]: Maybe update the JSON in the javadoc above?

Comment (rohansingh) [inline helios-client/src/main/java/com/spotify/helios/common/descriptors/HostInfo.java]: Done.
